clarifai/runners/dockerfile_template/Dockerfile.nim

```diff
@@ -0,0 +1,71 @@
+FROM nvcr.io/nim/meta/llama-3.1-8b-instruct:1.1.2 as build
+
+FROM gcr.io/distroless/python3-debian12:debug
```

Comment:
dont use debug for final

Reply:
I'm getting error when trying to run model non-root image `gcr.io/distroless/python3-debian12:nonroot-8701094b7fe8ff30d0777bbdfcc9a65caff6f40b`

```
INFO 11-19 13:21:02.337 ngc_injector.py:218] Preparing model workspace. This step might download additional files to run the model.
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:117] One or more errors fetching files:
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
[11-19 13:21:06.914 ERROR  nim_sdk::hub::repo  rust/nim-sdk/src/hub/repo.rs:119] I/O error Permission denied (os error 13)
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/opt/nim/llm/vllm_nvext/entrypoints/launch.py", line 99, in <module>
    main()
  File "/opt/nim/llm/vllm_nvext/entrypoints/launch.py", line 42, in main
    inference_env = prepare_environment()
  File "/opt/nim/llm/vllm_nvext/entrypoints/args.py", line 143, in prepare_environment
    engine_args, extracted_name = inject_ngc_hub(engine_args)
  File "/opt/nim/llm/vllm_nvext/hub/ngc_injector.py", line 220, in inject_ngc_hub
    cached = repo.get_all()
Exception: I/O error Permission denied (os error 13)

```

Reply:
Okk, will try it